Directory.Build.props

```diff
@@ -149,6 +151,7 @@
     <_runtimeOS Condition="'$(_runtimeOS)' == 'tizen.4.0.0'">ubuntu.14.04</_runtimeOS>
     <_runtimeOS Condition="'$(PortableBuild)' == 'true'">$(_portableOS)</_runtimeOS>
     <ToolRuntimeRID>$(_runtimeOS)-x64</ToolRuntimeRID>
+    <MicrosoftNetCoreIlasmPackageRuntimeId Condition="'$(MicrosoftNetCoreIlasmPackageRuntimeId)' == ''">$(ToolRuntimeRID)</MicrosoftNetCoreIlasmPackageRuntimeId>
```

Comment:
Updated to use `ToolRuntimeRID`... Noting that I don't think this works today for building on a 32-bit OS, is that a "known" limitation?

Reply:
I don't think we have any builds that work on a 32-bit OS.... Do we?  The only 32-bit builds that I know of are `win-x86` and `win-arm` (which we build on an x64 Windows machine) and `linux-arm` (which we build on an x64 Linux machine).

Reply:
I don't think we have any CI legs, but this was more a question of "Do we support building 32-bit CoreFX from 32-bit Windows?". If we don't, this is probably fine; otherwise, this should probably be updated to be "correct".

Reply:
I'm not sure if that is supported.  @weshaggard ?